### PR TITLE
LPD-16426 Create test for getting nested system objects when its title object field was deleted

### PR DESCRIPTION
**Pull request by devpedrotavares**
Follow up: https://github.com/brianchandotcom/liferay-portal/pull/150464

### CODE REVIEW

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/SystemObjectRelatedObjectEntriesTest.java [hunk 1]
```diff
@@ -334,6 +336,39 @@ public void testGetOneToManySystemObjectRelatedObjectEntries()
...
+
+		String titleObjectFieldValue = RandomTestUtil.randomString();
+
+		Map<String, Object> values = _userAccountJSONObject.toMap();
+
+		values.put(titleObjectField.getName(), titleObjectFieldValue);
+
+		_userSystemObjectDefinitionManager.updateBaseModel(
+			_userAccountJSONObject.getLong("id"), TestPropsValues.getUser(),
+			values);
```
Comment (carolmariaabb) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/SystemObjectRelatedObjectEntriesTest.java:358]: @devpedrotavares can we change it to:

```
_userSystemObjectDefinitionManager.updateBaseModel(
	_userAccountJSONObject.getLong("id"), TestPropsValues.getUser(),
	HashMapBuilder.<String, Object>put(
		titleObjectField.getName(), titleObjectFieldValue
	).putAll(
		_userAccountJSONObject.toMap()
	).build());

```
?

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by devpedrotavares** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**